src/openrct2/actions/GameAction.cpp

```diff
@@ -52,8 +52,21 @@ namespace GameActions
         return factory;
     }
 
+    void Initialize()
```

Comment:
@IntelOrca already mentioned in the chat this can be probably moved later into the Context